buildkite/scripts/rosetta-integration-tests.sh

```diff
@@ -52,12 +55,16 @@ echo "=========================== GENERATING GENESIS LEDGER FOR ${MINA_NETWORK}
 mkdir -p $MINA_KEYS_PATH
 mina-dev advanced generate-keypair --privkey-path $MINA_KEYS_PATH/block-producer.key
 mina-dev advanced generate-keypair --privkey-path $MINA_KEYS_PATH/snark-producer.key
+mina-dev advanced generate-keypair --privkey-path $MINA_KEYS_PATH/faucet.key
 chmod -R 0700 $MINA_KEYS_PATH
 BLOCK_PRODUCER_PK=$(cat $MINA_KEYS_PATH/block-producer.key.pub)
 SNARK_PRODUCER_PK=$(cat $MINA_KEYS_PATH/snark-producer.key.pub)
+FAUCET_PK=$(cat $MINA_KEYS_PATH/faucet.key.pub)
+FAUCET_SK=$(mina-dev advanced dump-keypair --privkey-path $MINA_KEYS_PATH/faucet.key | grep "Private key" | sed -e "s/Private key: //")
 
 mkdir -p $MINA_CONFIG_DIR/wallets/store
 cp $MINA_KEYS_PATH/block-producer.key $MINA_CONFIG_DIR/wallets/store/$BLOCK_PRODUCER_PK
+cp $MINA_KEYS_PATH/faucet.key $MINA_CONFIG_DIR/wallets/store/$FAUCET_PK
```

Comment:
Is this really necessary, provided we import those keys with `mina accounts import` command later? I think it should do the copy automatically for us.

Reply:
That's a good point. I saw on other scripts the keys being copied to that dir, but I don't actually know if that's needed. Nevertheless, I am not using a faucet anymore, so this line has been removed.

Reply:
The manual copy has been there initially on the the readmes until I learned about `mina accounts import` command, so maybe that's the source of confusion.